src/sparseml/pytorch/optim/modifier_quantization.py

```diff
@@ -278,6 +291,27 @@ def quantize_linear_activations(self) -> bool:
         """
         return self._quantize_linear_activations
 
+    @ModifierProp()
+    def enable_int4_activations(self) -> bool:
+        """
+        :return: if True, the quantization range for activations will be
+            reduced to 4 bit integers.
+        """
+        return self._enable_int4_activations
+
+    @ModifierProp()
+    def qconfig_activation_kwargs(self) -> Dict[str, Any]:
+        """
+        :return: Dictionary with correct quant_min, quant_max, and dtype values
+            for activations
+
+        """
+        return dict(
```

Comment:
this should not be computed dynamically. modifier props need to be creatable from recipe. (see above comments about adding a modifier prop for this and the weights